apps/api/src/server/applications/application/representations/representation.validators.js

```diff
@@ -3,7 +3,9 @@ import { body } from 'express-validator';
 import { validationErrorHandler } from '../../../middleware/error-handler.js';
 
 const isStrictTypeBoolean = async (value) => {
-	if (typeof value !== 'boolean') throw new TypeError(`Must be a boolean`);
+	if (typeof value !== 'boolean') {
+		if (value !== null) throw new TypeError(`Must be a boolean or null`);
+	}
```

Comment:
-could be done within one `If` by using `||` operator?
-also the name should be updated as is doesn't strictly check for booleans anymore

Reply:
or won't work but and will. 

Func name changed.